features/DocumentController.feature

```diff
@@ -262,15 +251,14 @@ Feature: Document Controller
       | "document-1" | { "name": "document1" } |
       | "document-2" | { "name": "document2" } |
       | "document-3" | { "name": "document3" } |
-    When I "get" the following document ids:
+    When I execute the "Exists" action on the following document ids:
```

Comment:
This should raise an error since the `document:Exists` action does not exists :thinking: 

Also, you don't need a new step, you can use this one 
```
    When I successfully execute the action "document":"mExits" with args:
      | body.ids[0]     | "id1"                      |
      | body.ids[2]     | "id1"                      |
      | body.ids[3]     | "id1"                      |
      | body.ids[4]     | "id1"                      |
```

Reply:
If you look in [document-steps.js](https://github.com/kuzzleio/kuzzle/pull/2265/files#diff-9f541c6eda56ab57543db808194ae15c2b9f448967279667815b2eed019e604bR67), I used the `sdk.query()` method.

Reply:
Ah I saw, the first line transform it into `mExists` so it was misleading to me.

Actually you don't need an extra step, you can just use a standard one. We should be careful to not add too many different steps because it increase maintainability